### PR TITLE
Add version to transaction

### PR DESCRIPTION
**Pull request by duanyytop**
add version to transaction and create contract_utils which contain some common methods

### CODE REVIEW

File: example/contract_utils.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
+const privkey = '352416e1c910e413768c51390dfd791b414212b7b4fe6b1a18f58007fa894214';
+const quota = 999999;
+
+var getTransactionReceipt = function(web3, hash, callback) {
+    // wait for receipt
+    var count = 0;
+    var filter = web3.eth.filter('latest', function(err){
+        if (!err) {
```
Comment (Keith-CY) [inline example/contract_utils.js]: Need Error Handler

File: example/contract_utils.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                    }
+                })
+            }
+        }
+    });
+}
+
+var initBlockNumber = function(web3, callback) {
+    web3.eth.getBlockNumber(function(err, res){
+        if (!err) {
```
Comment (Keith-CY) [inline example/contract_utils.js]: Need Error Handler

File: example/multi_solidity.js [hunk 3]
```diff
@@ -72,70 +53,27 @@ async function deployContract() {
...
-                        filter.stopWatching(function() {});
-                        const result = myContract.getMul.call();
-                        logger.info("get multi method result: " + JSON.stringify(result));
-                    }
-                });
-            }
-        }
-    });
+    contractUtils.getTransactionReceipt(web3, result.hash, function(receipt) {
+        const result = myContract.getMul.call();
```
Comment (Keith-CY) [inline example/multi_solidity.js]: Name Conflict with `var result =  myContract.setMul(4, 50, { ...commonParams });`

File: example/multi_solidity.js [hunk 4]
```diff
@@ -72,70 +53,27 @@ async function deployContract() {
...
-                        logger.info("get method add result: " + JSON.stringify(result));
-                    }
-                });
-            }
-        }
-    });
-}
-
+    contractUtils.getTransactionReceipt(web3, result.hash, function(receipt) {
+        const result = myContract.getAdd.call();
```
Comment (Keith-CY) [inline example/multi_solidity.js]: Name Conflict with `var result =  myContract.setAdd(29, 50, { ...commonParams });`

File: example/storage_test.js [hunk 5]
```diff
@@ -100,61 +81,18 @@ function getAbi(address) {
...
-                        const result = myContract.get.call();
-                        console.log("get method result: " + JSON.stringify(result));
-                    }
-                });
-            }
-        }
-    });
-
+    contractUtils.getTransactionReceipt(web3, result.hash, function(receipt) {
+        const result = myContract.get.call();
```
Comment (Keith-CY) [inline example/storage_test.js]: Name Conflict with `var result =  myContract.set(5, {`

File: example/contract_utils.js (hunk 1 above)
Comment (duanyytop) [reply example/contract_utils.js]: get

File: example/multi_solidity.js (hunk 3 above)
Comment (duanyytop) [reply example/multi_solidity.js]: get

**Pull request merged**